src/pdn/README.md

```diff
@@ -322,62 +347,42 @@ add_pdn_connect -grid rotated_rams -layers {metal4 metal6}
-| `-dont_add_pins` | Prevent the creation of block pins during |
-| `-reset` | Reset the grid and domain specifications |
-| `-ripup` | Ripup the existing power grid, as specified by the voltage domains |
-| `-report_only` | Print the current specifications |
-| `-failed_via_report` | Generate a report file which can be viewed in the DRC viewer for all the failed vias (ie. those that did not get built or were removed). |
-
-### Repairing power grid vias after detailed routing
+### Repairing Power Grid vias after Detailed Routing
 
 To remove vias which generate DRC violations after detailed placement and routing use `repair_pdn_vias`.
 
-```
-repair_pdn_vias [-all] \
-                [-net net_name]
+```tcl
+repair_pdn_vias 
+    [-all]
+    [-net net_name]
 ```
 
 #### Options
 
-| Name | Description |
+| Switch Name | Description |
 | ----- | ----- |
-| `-all` | Repair vias on all supply nets |
-| `-net` | Repair only vias on the specified net |
-
-### Converting former PDNGEN configuration file to tcl commands
+| `-all` | Repair vias on all supply nets. |
+| `-net` | Repair only vias on the specified net. |
 
-To get an initial conversion from the former PDNGEN configuration file to the current tcl commands use `convert_pdn_config`.
-This command will provide an initial set of commands based on the provided file, it is recommended that the user double-check
-the conversion to ensure nothing was missed.
+### Useful Developer Commands
 
-```
-convert_pdn_config config_file
-```
+If you are a developer, you might find these useful. More details can be found in the [source file](../src/PdnGen.cc) or the [swig file](PdnGen.i).
 
-#### Options
-
-| Name | Description |
+| Command Name | Description |
 | ----- | ----- |
-| `config_file` | Path to the old configuration file |
-
+| `name_cmp` | Compare 2 input strings `obj1` and `obj2` if they are equal. |
+| `check_design_state` | Check if design is loaded. |
+| `get_layer` | Get the layer reference of layer name. |
+| `get_voltage_domains` | Gets a Tcl list of power domains in design. |
+| `match_orientation` | Checks if a given orientation `orient` is within a list of orientations `orients`. |
+| `get_insts` | Get Tcl list of instances. |
+| `get_masters` | Get Tcl list of masters. |
+| `get_one_to_two` | If a Tcl list has one element `{x}`, Tcl list `{x x}` is returned. If a Tcl list of two elements `{y y}`, list as is returned. Otherwise, for any other list lengths, error is triggered. |
+| `get_one_to_four` | Similar logic for above function, except the logic only works for lists of length one, two and four respectively. All other list lengths triggers error. |
+| `get_obstructions` | Get Tcl list of layers. |
+| `get_starts_with` | If value starts with `POWER`, return 1; else if value starts with `GROUND` return 0; else return error. |
+| `get_mterm` | Find master terminal. |
+| `get_orientations` | Get list of valid orientations. | 
```

Comment:
```suggestion
| `name_cmp` | Compare 2 input strings `obj1` and `obj2` if they are equal. |
| `check_design_state` | Check if the design is loaded. |
| `get_layer` | Get the layer reference of the layer name. |
| `get_voltage_domains` | Gets a Tcl list of power domains in design. |
| `match_orientation` | Checks if a given orientation `orient` is within a list of orientations `orients`. |
| `get_insts` | Get the Tcl list of instances. |
| `get_masters` | Get the Tcl list of masters. |
| `get_one_to_two` | If a Tcl list has one element `{x}`, Tcl list `{x x}` is returned. If a Tcl list of two elements `{y y}`, list as is returned. Otherwise, for any other list lengths, an error is triggered. |
| `get_one_to_four` | Similar logic for the above function, except the logic only works for lists of length one, two, and four, respectively. All other list lengths trigger errors. |
| `get_obstructions` | Get the Tcl list of layers. |
| `get_starts_with` | If value starts with `POWER`, return 1; else if value starts with `GROUND` return 0; else return error. |
| `get_mterm` | Find master terminal. |
| `get_orientations` | Get the list of valid orientations. | 
```